src/rendering/text/Text.ts

```diff
@@ -49,30 +42,4 @@ export class Text extends Container<TextView>
     {
         return this.view.style;
     }
-
-    /**
-     * Destroys this text and optionally its texture and children.
-     * Do not use a Sprite after calling `destroy`.
-     * @param options - Options parameter. A boolean will act as if all options
-     *  have been set to that value
-     * @param {boolean} [options.children=false] - if set to true, all the children will have their destroy
-     *  method called as well. 'options' will be passed on to those calls.
-     * @param {boolean} [options.texture=false] - Should it destroy the current texture of the sprite as well
-     * @param {boolean} [options.textureSource=false] - Should it destroy the textureSource of the sprite as well
-     * @param {boolean} [options.context=false] - Only used for children with graphicsContexts e.g. Graphics.
-     * If options.children is set to true it should destroy the context of the child graphics
-     */
-    public destroy(options: DestroyOptions = false): void
-    {
-        if (typeof options === 'boolean')
-        {
-            options = { children: options };
-        }
-
-        options = Object.assign({}, defaultDestroyOptions, options);
-
-        super.destroy(options);
```

Comment:
We need to assign the default destroy options in the `TextView` now right?

Reply:
i don't think so any more - as it does not extend sprite anymore (eg no texture to manage!)